packages/code-du-travail-ui/src/__tests__/NoAnswer.test.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { render } from "react-testing-library";
+import NoAnswer from "../NoAnswer";
+import Button from "../Button";
+
+describe("<NoAnswer />", () => {
+  test("should render", () => {
+    const { container } = render(
+      <NoAnswer>
+        <Button primary onClick={jest.fn()}>
+          Feedback
+        </Button>
+      </NoAnswer>
+    );
+    expect(container).toMatchSnapshot();
```

Comment:
For small test and snapshot you can use `toMatchSnapshotInline` 👌

Reply:
thanks for the hint, I will do it in another pr